src/pages/detail/GroupMembersTab.tsx

```
@@ -1,7 +1,25 @@
+import { getUserInfo } from "@/apis/auth";
+import { UserInfo } from "os";
+import useSWR from "swr";
+
 interface GroupMembersTabProps {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)